support/metrics/metricsTracker.go

```diff
@@ -234,6 +264,9 @@ func (mt *MetricsTracker) SendUpdateEvent(now time.Time, success bool, millisFor
 		Success:                      success,
 		MillisForUpdate:              millisForUpdate,
 		SecondsSinceLastUpdateMetric: secondsSinceLastUpdateMetric,
+		NumPruneOps:                  mt.numOps.PruneOps,
```

Comment:
`SendUpdateEvent` should take in the struct `UpdateLoopResult` (defined in a comment below)

so here we would say:

```
Success: updateLoopResult.success,
NumPruneOps: updateLoopResult.numPruneOps,
NumUpdateOpsDelete: updateLoopResult.numUpdateOpsDelete,
NumUpdateOpsUpdate: updateLoopResult.numUpdateOpsUpdate,
```

makes sense?